.github/workflows/symfony-test.yml

```
@@ -0,0 +1,291 @@
+    strategy:
+      fail-fast: false
+      matrix:
+        projects: ${{ fromJSON(needs.define-matrix.outputs.projects) }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Download all workflow run artifacts
+        uses: actions/cache@v4
```

actionlint (1.7.4)

missing input "key" which is required by action "actions/cache@v4". all required inputs are "key", "path"

(action)